src/VirtoCommerce.ExperienceApiModule.Core/Index/IGetSingleDocumentQuery.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace VirtoCommerce.ExperienceApiModule.Core.Index
+{
+    public interface IGetSingleDocumentQuery : ISearchQuery, IHaveObjectId, IHasIncludeFields
```

Comment:
Do we need this interface in addition to `IGetAllDocumentsByIdsQuery`?
Isn't it just a specific case?